<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-14T10:44:41 IST" name="Suite" finished-at="2023-07-14T10:44:57 IST" duration-ms="15918">
    <groups>
    </groups>
    <test started-at="2023-07-14T10:44:41 IST" name="Errorvalidation" finished-at="2023-07-14T10:44:57 IST" duration-ms="15918">
      <class name="RahulshettyWebsiteMain.ErrorValidationTest">
        <test-method is-config="true" signature="lanchApplication()[pri:0, instance:RahulshettyWebsiteMain.ErrorValidationTest@29ca3d04]" started-at="2023-07-14T10:44:42 IST" name="lanchApplication" finished-at="2023-07-14T10:44:55 IST" duration-ms="13964" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lanchApplication -->
        <test-method signature="LoginErrorValidatonTest()[pri:0, instance:RahulshettyWebsiteMain.ErrorValidationTest@29ca3d04]" started-at="2023-07-14T10:44:55 IST" name="LoginErrorValidatonTest" finished-at="2023-07-14T10:44:56 IST" duration-ms="912" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrectd email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrectd email or password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at RahulshettyWebsiteMain.ErrorValidationTest.LoginErrorValidatonTest(ErrorValidationTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidatonTest -->
        <test-method is-config="true" signature="close()[pri:0, instance:RahulshettyWebsiteMain.ErrorValidationTest@29ca3d04]" started-at="2023-07-14T10:44:57 IST" name="close" finished-at="2023-07-14T10:44:57 IST" duration-ms="267" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- RahulshettyWebsiteMain.ErrorValidationTest -->
    </test> <!-- Errorvalidation -->
  </suite> <!-- Suite -->
</testng-results>
